src/styles/blocks/header.scss

```diff
@@ -0,0 +1,74 @@
+@use '../utils/variables' as *;
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  font-weight: 500;
+
+  &__logo {
+    align-self: center;
+    display: flex;
+  }
+}
+
+.navigation {
+  display: flex;
+  align-items: center;
+}
+
+.nav {
+  &_list {
+    display: flex;
+    list-style: none;
+    margin: 0;
+    padding-left: 0;
+  }
+
+  &_item {
+    display: flex;
+  }
+
+  &_link {
+    color: $black-color;
+    text-decoration: none;
+    font-size: 12px;
+    text-transform: uppercase;
+    align-items: center;
+    display: flex;
+    height: 60px;
+    margin-left: 20px;
+
+    &:visited {
+      color: $black-color;
+    }
+
+    &:hover {
+      color: $blue-color;
+    }
+
+    &:active {
+      color: $blue-color;
+      position: relative;
+    }
+
+    &--is-active {
+      &:first-child {
+        color: $blue-color;
+        position: relative;
+        margin-left: 0;
+      }
```

Comment:
The `:first-child` pseudo-class is used within the `&--is-active` class. This might not work as intended if the active link is not the first child. Consider using a different approach to ensure the active link styling applies correctly regardless of its position.